util/lua/lua.go

```diff
@@ -346,22 +351,38 @@ func noAvailableActions(jsonBytes []byte) bool {
 	return string(jsonBytes) == "[]"
 }
 
-func (vm VM) GetResourceActionDiscovery(obj *unstructured.Unstructured) (string, error) {
+func (vm VM) GetResourceActionDiscovery(obj *unstructured.Unstructured) ([]string, error) {
 	key := GetConfigMapKey(obj.GroupVersionKind())
+	var discoveryScripts []string
+
 	override, ok := vm.ResourceOverrides[key]
 	if ok && override.Actions != "" {
 		actions, err := override.GetActions()
 		if err != nil {
-			return "", err
+			return nil, err
+		}
+		if actions.AddBuildInActions {
+			discoveryScripts = append(discoveryScripts, actions.ActionDiscoveryLua)
+		} else {
```

Comment:
Instead of adding both actions in a list, when merge is set, load the default one and add logic to override the default with the custom one if the name match

Reply:
i don't  want to parse the lua script in  `GetResourceActionDiscovery` function,  instead use the `ExecuteResourceActionDiscovery`  which already parse the scripts to to evaluate the actions.

Reply:
Makes sense. I got confused between `ExecuteResourceAction` and `ExecuteResourceActionDiscovery` 👍